src/main/java/com/fakng/fakngagrgtr/parser/apple/AppleParser.java

```diff
@@ -0,0 +1,145 @@
+package com.fakng.fakngagrgtr.parser.apple;
```

Comment:
Please, add apple-sample.json file for future reference to JSON you're parsing here. See how it's done for Google parser, please